src/mach.rs

```diff
@@ -630,7 +632,14 @@ fn build_relocations(artifact: &Artifact, symtab: &SymbolTable) -> Relocations {
                     (_, &Decl::DataImport) => (false, X86_64_RELOC_GOT_LOAD),
                 }
             }
-            Reloc::Raw { .. } => panic!("unimplemented Reloc::Raw"),
+            Reloc::Raw { reloc, addend } => {
+                debug_assert!(reloc <= u8::max_value() as u32);
+                assert!(addend == 0);
```

Comment:
I am not really sure about which assertion to make a debug assertion or not.

Reply:
I would have probably made them both asserts if they’re checking fundamental programmer error, but dunno

Reply:
`reloc` will get truncated to an `u8`. Cranelift uses the constants from `goblin`, which are `u8`, so it can't overflow. I expect other users of the `link_with` api to use those constants too, so that assertion is unlikely to fail. This makes it a waste of cycles to have in release mode.